### PR TITLE
Various fixes to sword JSON

### PR DESCRIPTION
**Pull request by ToxiClay**
#### Summary

SUMMARY: None

#### Purpose of change

The purpose of this PR is to (correctly!) propose changes to the `swords_and_blades` JSON file.

#### Describe the solution

I corrected various errors, and this time did _not_ touch restricted files.

#### Describe alternatives you've considered

No alternatives.

#### Testing

Linting performed.

### CODE REVIEW

**Review by mlangsdorf — APPROVED** [review]

File: data/json/items/melee/swords_and_blades.json [hunk 1]
```diff
@@ -180,7 +180,7 @@
     "type": "TOOL",
     "category": "weapons",
     "name": { "str": "RM42 fighting knife", "str_pl": "RM42 fighting knives" },
-    "description": "This sturdy matte black Rivtech combat dagger features a long and slim double-edged blade with a spear-point and a distinctive slip-resistant grip which can also be used to affix it to a suitable firearm.  Originally manufactured for the military, it was very popular in films and among collectors due to its fearsome appearance.",
+    "description": "This sturdy matte black Rivtech combat dagger features a long, slim double-edged blade with a spear point and a distinctive slip-resistant grip, which can also be used to affix it to a suitable firearm.  Originally manufactured for the military, it was very popular in films and among collectors due to its fearsome appearance.",
```
Comment (actual-nh) [inline data/json/items/melee/swords_and_blades.json:183]:
```suggestion
    "description": "This sturdy, matte-black Rivtech combat dagger features a long, slim double-edged blade with a spear point and a distinctive slip-resistant grip, which can also be used to affix it to a suitable firearm as a bayonet.  Originally manufactured for the military, it was very popular in films and among collectors due to its fearsome appearance.",
```

File: data/json/items/melee/swords_and_blades.json [hunk 2]
```diff
@@ -1509,7 +1509,7 @@
     "symbol": "/",
     "color": "light_gray",
     "name": { "str": "rapier" },
-    "description": "This is a blunted, cheap replica of a thin sword with an ornate hand guard.  It looks like the preferred weapon of gentlemen and swashbucklers.  Light and quick, it makes any battle a stylish battle.",
+    "description": "This is a blunted, cheap replica of a thin sword with an ornate handguard.  It looks like the preferred weapon of gentlemen and swashbucklers.  Light and quick, it makes any battle a stylish battle.",
```
Comment (actual-nh) [inline data/json/items/melee/swords_and_blades.json:1512]:
```suggestion
    "description": "This is a blunted, cheap replica of a thin sword with an ornate handguard.  It looks like the preferred weapon of gentlemen and swashbucklers.  Light and quick, it makes any battle stylish.",
```

**Review by actual-nh — COMMENTED** [review]
Good!

**Comment by actual-nh** [comment]
> I corrected various errors, and this time did _not_ touch restricted files.

Restricted files?

File: data/json/items/melee/swords_and_blades.json (hunk 1 above)
Comment (ToxiClay) [reply data/json/items/melee/swords_and_blades.json:183]: I thought about this, but then I considered that I wouldn't hyphenate "glossy white" in similar circumstances.

Comment (ToxiClay) [reply data/json/items/melee/swords_and_blades.json:183]: I'll manually commit "as a bayonet," though.

**Comment by ToxiClay** [comment]
> > I corrected various errors, and this time did _not_ touch restricted files.
> 
> Restricted files?

I was instructed not to edit the files in the `lang` directory by hand. That's what I meant by 'restricted.'

File: data/json/items/melee/swords_and_blades.json (hunk 2 above)
Comment (ToxiClay) [reply data/json/items/melee/swords_and_blades.json:1512]: I think the repetition here serves well. If you imagine the word 'stylish' being italicized if this were presented in prose, I think it works better.

File: data/json/items/melee/swords_and_blades.json (hunk 1 above)
Comment (actual-nh) [reply data/json/items/melee/swords_and_blades.json]: I think I would hyphenate "glossy white", actually, if it were together with anything else. But I probably tend to over-hyphenate; no problem. Comma after "sturdy"?

**Comment by actual-nh** [comment]
> I was instructed not to edit the files in the `lang` directory by hand. That's what I meant by 'restricted.'

Yeah, the translation setup is rather complicated (and, as far as I understand, under revision...).

**Pull request merged**